### PR TITLE
Add best pratice around operation counters

### PR DESCRIPTION
**Pull request by discordianfish**
After asking the same questions a dozen times, I thought we should add it to the docs.

Not sure about words and everything but better than nothing. Let me know if I should rephase/move.

### CODE REVIEW

**Comment by fabxc** [comment]
:+1:

File: content/docs/practices/instrumentation.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ this category.
 The key metrics in such a system are the number of performed queries, errors,
 and latency. The number of in-progress requests can also be useful.
 
+When exposing operation counters, create seperate metrics for total and failed number of
+requests. That makes calculating percental error rates easier.
```
Comment (brian-brazil) [inline content/docs/practices/instrumentation.md]: This doesn't really belong here, and is already present further down the page.

Ratios are also preferred to percentages.

Comment (discordianfish) [inline content/docs/practices/instrumentation.md]: Now I found it. Is Failures as Subsystem? Maybe that can be improved.
And maybe it should be made more explicit. As I said, I still get confused why not use labels but separate metrics and the docs don't really clarify that (to be fair, neither does my change.)

Comment (beorn7) [inline content/docs/practices/instrumentation.md]: I suggest to add the following at the end of the section (after the paragraph about counting queries):

For counting failed queries, see section [Failures](#failures) below.

File: content/docs/practices/instrumentation.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ this category.
 The key metrics in such a system are the number of performed queries, errors,
 and latency. The number of in-progress requests can also be useful.
 
+When exposing operation counters, create seperate metrics for total and failed number of
```
Comment (juliusv) [inline content/docs/practices/instrumentation.md]: s/seperate/separate/

**Comment by discordianfish** [comment]
Ok, so I removed the section again and just pointed to the bits in 'Failures'. That should be enough I guess.

**Comment by beorn7** [comment]
:+1:

**Pull request merged**